src/rust/ide/lib/parser/tests/parsing.rs

```diff
@@ -49,7 +53,7 @@ fn roundtrip_program(program:&str) {
 // === Metadata ===
 // ================
 
-/// Wrapper for using ant serializable type as metadata.
+/// Wrapper for using any serializable type as metadata.
```

Comment:
I think it was correct. We were serializing ants here, weren't we?

Reply:
Ants were not serialized. However, as our profiling proved, they are extensively used for transporting data between JS and WASM.